src/components/User/KycRequests/KycRequest.List.vue

```diff
@@ -234,11 +244,22 @@ export default {
         this.view.scrollPosition = 0
       })
     },
+
+    async getAccountIdByEmail () {
+      let address
+      try {
+        address = await api.users.getAccountIdByEmail(this.filters.email)
```

Comment:
should be fetched from new 'v3/identities/' endpoint. the current is no supported anymore